web_form_banner/static/src/js/web_form_banner.esm.js

```diff
@@ -0,0 +1,216 @@
+odoo.define("web_form_banner.save_plus_load", function (require) {
+    var rpc = require("web.rpc");
+    var FormController = require("web.FormController");
+
+    var root = function (ctrl) {
+        return (ctrl && (ctrl.el || (ctrl.$el && ctrl.$el[0]))) || null;
+    };
+    var alive = function (ctrl) {
+        var r = root(ctrl);
+        return (
+            r &&
+            r.isConnected &&
+            !(typeof ctrl.isDestroyed === "function" && ctrl.isDestroyed())
+        );
+    };
+    var qsa = function (el, sel) {
+        return Array.from(el ? el.querySelectorAll(sel) : []);
+    };
+    var first = function () {
+        for (var i = 0; i < arguments.length; i++) {
+            var v = arguments[i];
+            if (v !== null && v !== "") return v;
+        }
+        return null;
+    };
+    var childSpan = function (el) {
+        if (!el) return null;
+        if (el.querySelector) {
+            return el.querySelector(":scope > span") || null;
+        }
+        var c = el.firstElementChild;
+        return c && c.tagName === "SPAN" ? c : null;
+    };
+    var after = function (p, fn) {
+        if (p && typeof p.always === "function") {
+            p.always(fn);
+            return p;
+        }
+        return Promise.resolve(p).finally(fn);
+    };
+    var shrinkDraft = function (d) {
+        return Object.entries(d || {}).reduce(function (o, kv) {
+            var k = kv[0],
+                v = kv[1],
+                t = typeof v;
+            if (v === null || t === "string" || t === "number" || t === "boolean") {
```

Comment:
This change should be reverted.

```suggestion
            if (v == null || t === "string" || t === "number" || t === "boolean") {
```